styles.css

```diff
@@ -0,0 +1,29 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  background-color: rebeccapurple;
+  overflow: hidden;
+}
+
+header h1 {
+  color: powderblue;
+  padding: 15px;
+  text-shadow: -1px 0 hotpink, 0 1px hotpink, 1px 0 hotpink, 0 -1px hotpink;
+}
+
+h2 {
+  margin: 0;
+  margin: 50px 0;
+  border: 2px solid black;
```

Comment:
Maybe a little padding to the left would give the h2 some room to breathe.

Reply:
Good catch, done!